api/filter/accesslog.proto

```diff
@@ -95,16 +258,63 @@ message FileAccessLog {
   string format = 2;
 }
 
+// Configuration for the built-in "envoy.grpc_access_log" access log type.
+message AccessLogServiceConfig {
+  // The friendly name of the access log to be returned in StreamAccessLogsMessage.Prefix. This
+  // allows the access log server to differentiate between different access logs coming from the
+  // same Envoy.
+  string log_name = 1;
+}
+
+// Stream message for the StreamAccessLogs API. Envoy will open a stream to the server and stream
+// access logs without ever expecting a response.
+message StreamAccessLogsMessage {
+  message Prefix {
+    // The node sending the access log messages over the stream.
+    Node node = 1;
+
+    // The friendly name of the log configured in AccessLogServiceConfig.
+    string log_name = 2;
+  }
+
+  // Prefix data that will only be sent in the first message on the stream. This is effectively
+  // structured metadata and is a performance optimization.
+  Prefix prefix = 1;
```

Comment:
It might make sense to make this less stream oriented in terminology, and call this something like `Identifier`. I'm thinking of a scenario in which we have on the wire or in some stored proto a collection of log messages from different Envoys, in which we attached identification information. This could be done with what you have there right now, but it feels like a stretch in terms of naming.

Reply:
Sure will change.